setup.py

```diff
@@ -30,7 +30,7 @@
 
 setup(
     name="mantarray_waveform_analysis",
-    version="0.9.0",
+    version="1.0.0",
```

Comment:
bumped major version because this PR changes the `find_twitch_indices` function args and is part of the public interface